src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/EmbeddingProviderFactory.java

```diff
@@ -40,8 +40,9 @@ EmbeddingProvider create(
           Map.entry(ProviderConstants.VERTEXAI, VertexAIEmbeddingClient::new),
           Map.entry(ProviderConstants.VOYAGE_AI, VoyageAIEmbeddingClient::new),
           Map.entry(ProviderConstants.COHERE, CohereEmbeddingClient::new),
+          Map.entry(ProviderConstants.MISTRAL, MistralEmbeddingClient::new),
           Map.entry(ProviderConstants.NVIDIA, NvidiaEmbeddingClient::new),
-          Map.entry(ProviderConstants.MISTRAL, MistralEmbeddingClient::new));
+          Map.entry(ProviderConstants.JINA_AI, JinaAIEmbeddingClient::new));
```

Comment:
Maybe add this in alphabetic order as well? (before Mistral)

Reply:
NP.